.circleci/config/base.yml

```diff
@@ -1247,7 +1181,7 @@ commands:
           command: |
             cd src
             if [ "$SKIP_DIST_ZIP" != "1" ]; then
-              ninja -C out/Default electron:electron_dist_zip << parameters.additional-targets >>
+              ninja -C out/Default electron:electron_dist_zip << parameters.additional-targets >> -j $NUMBER_OF_NINJA_PROCESSES
```

Comment:
maybe use `autoninja`?

Reply:
Tbh this change shouldn't really be here, I was just annoyed by this step going slower than it needed to